css/frame.css

```diff
@@ -1081,16 +1081,18 @@ button#editor-chart-button {
 .only-pro-content a {
 	display: inline-block;
 	padding: 7px 12px;
-	color: #fff;
-	background: #ff595e;
+	color: #fff !important;
```

Comment:
Is there something that interferes with our CSS, which requires the use of `!important`?

Reply:
Yup, it is this class: https://vertis.d.pr/i/kUFS1X. It was overriding the text color even before this changes.

Reply:
You can use a more extended selector like `.only-pro-content .only-pro-container a`. There is no need to be minimalist with the selector length

Reply:
That's a change that might introduce unwanted bugs, because we limit the styling to a more narrow class than how it was until now. 

Or where you referring to just the part with `!important` to be moved to the selector you mentioned ?

Reply:
It can also be only the part with `!important`.

Also, you can try this type of selector: `:is(.only-pro-content .only-pro-container, .only-pro-content ) a` which covers a wide range but also has a condition for edge cases.

Reply:
Changed them as you mentioned. Thanks for pointing it out!